spec/filewatch/reading_spec.rb

```diff
@@ -1,8 +1,10 @@
-
+# encoding: utf-8
 require 'stud/temporary'
 require_relative 'spec_helper'
 require 'filewatch/observing_read'
 
+LogStash::Logging::Logger::configure_logging("WARN")
```

Comment:
I think this should be done in a safer way like in a `before(:all)` that stores the current value and restores it in a `after(:all)` block

Reply:
Probably.
I should comment it out - its not pertinent to the tests themselves only to the human watching the test run.
Should we make it an env setting?
```
LOG_AT=WARN bundle exec rspec -fd ./spec
LOG_AT=DEBUG bundle exec rspec -fd ./spec
```

Reply:
if devutils is already disabling logging then disregard my comment

Reply:
the `LogStash::Logging::Logger::configure_logging("WARN")` instruction in the spec is after devutils does its bit.